Source/StateTester.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * 
+ */
+package cs505.group1.state;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * @author CS505 Fall17 Group1
+ */
+public class StateTester {
+
```

Comment:
Note there is no requirement for testing in this past Sunday's submission, but there is for Sprint 1.  However, please be aware this doesn't provide anything in terms of automated testing.  Make sure you make appropriate use of JUnit assertions here to make these test automated.

Reply:
Understood.